chart/compass/charts/ias-adapter/templates/rbac.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  {{- if eq .Values.global.database.embedded.enabled false }}
+  annotations:
+    iam.gke.io/gcp-service-account: {{ $.Values.global.database.sqlProxyServiceAccount | quote }}
```

Comment:
When we have this gcp annotation that means we're using the workload identity functionality from GCP. And the other half of it is to maintain/update the `proxy-user` service account in GCP. For that, a PR in the infra repo is needed. Valid for all environments and also for all k8s SA that use such annotation.